django_project/certification/migrations/0014_auto_20171114_0947.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.conf import settings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('certification', '0013_auto_20171014_1045'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
Hi @sonlinux , I think you should pull the latest code in develop and try to generate the migrations again. This migration does not seem right as it removes the degree and title field in the course convener model

Reply:
Hi @ann26 let me do so.

Reply:
remember to undo this migration before deleting it, otherwise you'll end up with an out of sync DB.